src/Builder.php

```diff
@@ -18,6 +18,10 @@
  */
 class Builder
 {
+    // Some configuration options are passed around in an array, using these
+    // keys
+    const CONF_EXPECT_JS_COOKIE  = 'expectJsCookie';
```

Comment:
It's applicable to `POST /verify` and `POST /events`, but not `GET /events`. It's a bit messy yeah, but wasn't sure of an alternative offhand.